### PR TITLE
Fix Verify Email Code method

### PR DESCRIPTION
**Pull request by jonatascastro12**
## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

**Comment by jonatascastro12** [comment]
Current dependencies on/for this PR:
* users
  * **PR #209** <a href="https://app.graphite.dev/github/pr/workos/workos-python/209" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/workos/workos-python/209?utm_source=stack-comment).

File: workos/users.py [hunk 1]
```diff
@@ -503,14 +503,14 @@ def verify_email(
...
-            USER_VERIFY_EMAIL_PATH,
+            USER_VERIFY_EMAIL_CODE_PATH,
             method=REQUEST_METHOD_POST,
             headers=headers,
             params=payload,
             token=workos.api_key,
         )
 
-        return WorkOSUser.construct_from_response(response).to_dict()
+        return WorkOSUser.construct_from_response(response["user"]).to_dict()
```
Comment (ameesha) [inline workos/users.py:513]: just to make sure we want to hard fail if for some reason the "user" key is missing right? (else we'd do .get('user', {})

**Review by ameesha — APPROVED** [review]

File: workos/users.py (hunk 1 above)
Comment (jonatascastro12) [reply workos/users.py:513]: Yeah... Since this is our API response, I think it's better than having a silent issue that may be hard to debug/understand.

**Pull request merged**